[2022-08-03 15:42:36,395] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:41:44.992108+00:00 [queued]>
[2022-08-03 15:42:36,614] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:41:44.992108+00:00 [queued]>
[2022-08-03 15:42:36,633] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:42:36,647] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 15:42:36,660] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:42:36,786] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load-data> on 2022-08-03 15:41:44.992108+00:00
[2022-08-03 15:42:36,853] {standard_task_runner.py:52} INFO - Started process 1059 to run task
[2022-08-03 15:42:36,911] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'load-data', 'manual__2022-08-03T15:41:44.992108+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmp6j25foa1', '--error-file', '/tmp/tmpriw0z89n']
[2022-08-03 15:42:36,924] {standard_task_runner.py:80} INFO - Job 45: Subtask load-data
[2022-08-03 15:42:37,589] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:41:44.992108+00:00 [running]> on host 3b0f05b8acbc
[2022-08-03 15:42:38,255] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=load-data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T15:41:44.992108+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T15:41:44.992108+00:00
[2022-08-03 15:42:38,354] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-08-03 15:42:38,399] {dbapi.py:231} INFO - Running statement: 
            COPY vehicles (id, vin, price, miles, stock_no, year, make, model, trim, body_type, vehicle_type, drivetrain, transmission, fuel_type, engine_size, engine_block, seller_name, street, city, state, zip)
            FROM PROGRAM 'gzip -dc ../files/vehicle_challenge.csv.gz' DELIMITER ',' CSV HEADER;, parameters: None
[2022-08-03 15:42:38,618] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 211, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 235, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.ExternalRoutineException: program "gzip -dc ../files/vehicle_challenge.csv.gz" failed
DETAIL:  child process exited with exit code 1

[2022-08-03 15:42:38,738] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=ltv-pipeline, task_id=load-data, execution_date=20220803T154144, start_date=20220803T154236, end_date=20220803T154238
[2022-08-03 15:42:38,815] {standard_task_runner.py:97} ERROR - Failed to execute job 45 for task load-data (program "gzip -dc ../files/vehicle_challenge.csv.gz" failed
DETAIL:  child process exited with exit code 1
; 1059)
[2022-08-03 15:42:38,912] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-03 15:42:39,122] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
